upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -111,6 +111,39 @@ There you can find links to upgrade notes for other versions too.
     -           ->setExpectedStatusCode(404);
     +           ->setExpectedStatusCode(302);
     ```
+- contact form has been moved to separate page. You can find the whole new setting in administration (`/admin/contact-form/`),
+  where you can edit main text for contact form. ([#1522](https://github.com/shopsys/shopsys/pull/1522))
+  
+  There are few steps need to be done, to make contact form work on FE
+    - in `ContactFormController` change previous `indexAction` with [the new one](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Controller/Front/ContactFormController.php). Please pay attention if you have some modification in previous implementation.
+    - in `ContactFormController` remove action `sendAction()`, it is not needed anymore
+    - remove `src/Resources/scripts/frontend/contactForm.js`, it is not needed anymore
+    - new localized route `front_contact` has been introduced with slug `contact`. This slug can be already in use in your project, because
+      previous SSFW versions have an article called `Contact` which was used as the contact page. Please move the article's content to the new contact page and completely remove the article.
+      To remove the article, please create [new migration](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Migrations/Version20191121171000.php) in you project which removes the article and its slug.
+      
+      !!! hint
```

Comment:
admonitions can be used only inside `docs/` folder from which the documentation is generated. 
Upgrades are still meant to be viewed from Github because it's nearly impossible to make lists with code snippets in mkdocs (tool to generate documentation) :)

Reply:
didn't know that. Removed.

Reply:
are you sure it's removed? :)

Reply:
I see it now :)